.circleci/config.yml

```diff
@@ -2254,6 +2254,13 @@ jobs:
             python ../build_package.py deb
             python ../build_package.py rpm
 
+            dpkg -X "$(ls *.deb)" /tmp/deb_package
+            warnings=$(dpkg-parsechangelog --file /tmp/deb_package/usr/share/doc/scalyr-agent-2/changelog.gz 2>&1 1>/dev/null | grep warning || true)
```

Comment:
I wonder if we should use ``set +e`` for this section and then check exit code of the command where we grep for "warning" and in case it's non-zero exit with 1 and call the command again to print actual warnings.

The issue with current approach is that it won't fail in case ``/tmp/deb_package/usr/share/doc/scalyr-agent-2/changelog.gz`` file doesn't exist.

Which means that check could be incorrectly passing in situations where that file doesn't exist for whatever reason.